### PR TITLE
fixed sles12-GM build

### PR DESCRIPTION
**Pull request by schubi2**

### CODE REVIEW

File: build_iso/sles12.rb [hunk 1]
```diff
@@ -65,12 +72,13 @@
...
 
   puts "\n**** Copy new ISO image to veewee/vagrant environment ****"
   puts "\n     destination: #{testing_iso}"
   FileUtils.cp("testing.iso", testing_iso)
 end
 
-puts "\n**** Cleanup ****"
-system("rm -rf #{cache_dir+'/*'}")
+#puts "\n**** Cleanup ****"
+#system("rm -rf #{cache_dir+'/*'}")
```
Comment (imobach) [inline build_iso/sles12.rb]: Is this dead code?

File: build_iso/sles12.rb [hunk 2]
```diff
@@ -65,12 +72,13 @@
   system "find . -name \"*.rpm\"|xargs mkdud -c #{version}.dud -d sle12 -i instsys,repo --prefix=37"
 
   puts "\n**** Creating new ISO image with the updated packages ****"
-  system "sudo mksusecd -c testing.iso --initrd=#{version}.dud #{iso_path}"
+puts "sudo mksusecd -c testing.iso --initrd=#{version}.dud #{iso_path} #{boot_dir}"
+  system "sudo mksusecd -c testing.iso --initrd=#{version}.dud #{iso_path} #{boot_dir}"
```
Comment (imobach) [inline build_iso/sles12.rb]: I'd just use a variable to store the whole command and then I will call `puts` and `system` just passing that variable (so we won't forget to update both lines when something changes).

**Comment by imobach** [comment]
LGTM

File: build_iso/sles12.rb (hunk 1 above)
Comment (schubi2) [inline build_iso/sles12.rb]: ups, you are right

File: build_iso/sles12.rb (hunk 2 above)
Comment (schubi2) [inline build_iso/sles12.rb]: It was just for debugging. I have removed.

**Pull request merged**